test/build/bundle_test.js

```diff
@@ -97,6 +97,18 @@ suite('Bundler', () => {
     return link != null;
   };
 
+  suite('urlFromPath()', () => {
+
+    test.skip('creates a URL path relative to root when called with a file path', () => setupTest({
```

Comment:
Our testing setup currently can't test path-related behavior on Windows. My next task will be to make this test and the several under it runnable.